packages/frontend/src/widgets/long-booking-modal/ui/LongBookingModal.tsx

```diff
@@ -0,0 +1,320 @@
+import { useState } from 'react'
+import Image from 'next/image'
+import { useForm, SubmitHandler, Controller } from 'react-hook-form'
+import classNames from 'classnames'
+
+import {
+  ButtonGeneral,
+  DetailsLong,
+  ModalBase,
+  ChipBox,
+  PriceLong,
+  LinkGeneral,
+  SwitchBase
+} from '@shared/ui'
+
+import { MetroInfo } from '@entities/MetroInfo'
+import { AdressInfo } from '@entities/AdressInfo'
+
+import styles from './LongBookingModal.module.scss'
+
+interface IApartment {
+  price: string
+  image_src: string
+  roomNum: number
+  area: number
+  storey: number
+  totalStoreys: number
+  street: string
+  metro: {
+    name: string
+    timeFoot: number
+    timeVehicle: number
+  }
+}
+
+interface ILongBookingModalProps {
+  apartmentInfo: IApartment
+  onClose: () => void
+  isOpen: boolean
+}
+
+interface IBookingForm {
+  phoneNumber: string
+  nickname: string
+  comment: string
+  location: string
+  checkbox: boolean
+  additionally: []
+}
+
+const chips = {
+  additionallyFirst: ['Иностранцы', 'Семья', 'Более 4х человек', 'Студенты'],
+  additionallySecond: [
+    'Есть дети',
+    'Есть собака',
+    'Есть котик',
+    'Оплата от юр.лица'
+  ],
+  additionallyThird: ['Хочу снять квартиру на пару месяцев', 'Онлайн-просмотр']
+}
+
+const defaultValues: any = {
+  additionally: []
+}
+
+export const LongBookingModal: React.FC<ILongBookingModalProps> = ({ apartmentInfo, isOpen, onClose }: ILongBookingModalProps) => {
```

Comment:
Импортируй FC из React и тогда можно так:
```suggestion
export const LongBookingModal: FC<ILongBookingModalProps> = ({ apartmentInfo, isOpen, onClose }: ILongBookingModalProps) => {
```

Reply:
сделано